charts/fluentd/README.md

```diff
@@ -4,6 +4,39 @@
 
 ## Chart Values
 
-```sh
-helm show values fluent/fluentd
-```
```

Comment:
I think `helm show values fluent/fluentd` is good as it deduplicates the declaration of values in the README which is already in the values file, more and more charts are putting this in the readme.